.github/workflows/go.yml

```diff
@@ -164,6 +164,8 @@ jobs:
       run: ./ci/check-codegen.sh
     - name: Check manifest
       run: ./ci/check-manifest.sh
+    - name: Check copyright
+    run: ./ci/check-copyright.sh
```

Comment:
I notice that there is a formatting issue here. Could it be why the CI jobs are not running for your PR (I notice that the jobs not running are the ones defined in this file)?